### ISSUE TITLE
Some recommended part for user in searched page

### ISSUE THREAD

**Issue by Aakash-DD** [fossasia/susper.com#1421]
**I'm submitting a ...**

- [ ] feature request

**Current behavior:**

Does not show some recommended part for user
**Expected behavior:**

May show a div (part) for Most searched Questions on Susper
- Can be shown as recommended part for the user

**Screenshots of the issue:**

![screenshot 48 _li](https://user-images.githubusercontent.com/33255535/51556515-69321000-1ea0-11e9-93bf-20b6b72ea96e.jpg)

**Will work on the issue:**
Someone else can be assigned :)

**Comment by Aakash-DD** [comment]
Depending on the user search, some related or similar things can be shown as a part of recommendation.
If this will be there, user will try to explore more, this will be a great attention and impress users as well.

**Comment by praveenojha33** [comment]
@Aakash-DD Yeah It will be great if you can implement this feature using YaCy only.
